camayoc/tests/qpc/cli/test_sources.py

```diff
@@ -286,6 +284,304 @@ def test_add_with_port_negative(
     assert qpc_source_add.exitstatus == 2
 
 
+# @pytest.mark.parametrize('source_type', QPC_HOST_MANAGER_TYPES)
```

Comment:
I will update the test once quipucords is fixed to not accept SSL options for network sources.